packages/forma-36-react-components/src/components/Card/Card.tsx

```diff
@@ -55,48 +50,44 @@ const defaultProps: Partial<CardPropTypes> = {
   selected: false,
 };
 
-export class Card extends Component<CardPropTypes> {
-  static defaultProps = defaultProps;
+export const Card: React.FC<CardPropTypes> = ({
```

Comment:
I'm not a big fan of using this `FC` type as it leads to implicit props being included on the component that we might not want in all cases, for instance `children`. But if we do it, we should agree to do it for all our components 🙂 What do you think?

Reply:
Oohhhhh I never thought about that... you are right!

I'm not attached to `FC` tbh. It's just the way I'm used to writing components by now hahaha
And now that you mentioned the implicit props, I agree we should use another type as a convention
Should we declare `children` in the Props interface every time a component uses it?

Reply:
> Should we declare children in the Props interface every time a component uses it?

That would be my preference at least. I'd rather add a `children` prop when children are used than delete it from the implicit props when they aren't used 😅